### DISCUSSION TITLE
How do I set the max size of the room programmatically?

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#3196] [General Questions & Discussion] [answered]
I'm interested in changing the max size of the room programmatically and setting it in the createAndRedirectToNewHub function (when a new room is created). 

I can see in the UI on room-settings-ui.tsx where the UI is, but I don't see where it connects to. 

Can someone point me in the right direction and if it's even possible to set the max number of users as the room is being created?

Thanks a lot

**Comment by robinkwilson** [comment, marked as answer]
It's in the works! You'll be able to when the new APIs are published. Here's the PR to follow (and the line you want): https://github.com/mozilla/reticulum/pull/399/files#diff-efd2e6624ac2d84bd7bdf162c0b8ab6459b1ee871266de4d97c451c20a93635eR185

I'd personally wait until this is published because this is the official way of doing it. However, if you needed an immediate fix, you can take a peek at how the Room Settings are updated via websocket in the code. Again I'd recommend waiting for the APIs to be published.

**Reply by ybinstock** [reply]
Thanks Robin, 

I do need to figure out a solution in the next 2-3 weeks, so I'll look into your idea about how the room settings are set with websocket.
